S/SuiteSparse/SuiteSparse/build_tarballs.jl

```diff
@@ -2,57 +2,48 @@ include("../common.jl")
 
 name = "SuiteSparse"
 
-sources = [
-    sources;
-    DirectorySource("./bundled")
-]
-
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/SuiteSparse
+apk add --upgrade cmake --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main
 
-# Apply Jameson's shlib patch
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/SuiteSparse-shlib.patch
+cd $WORKSPACE/srcdir/SuiteSparse
 
 # Disable OpenMP as it will probably interfere with blas threads and Julia threads
 FLAGS+=(INSTALL="${prefix}" INSTALL_LIB="${libdir}" INSTALL_INCLUDE="${prefix}/include" CFOPENMP=)
-
 if [[ ${bb_full_target} == *-sanitize+memory* ]]; then
     # Install msan runtime (for clang)
     cp -rL ${libdir}/linux/* /opt/x86_64-linux-musl/lib/clang/*/lib/linux/
 fi
 
-if [[ ${target} == *mingw32* ]]; then
-    FLAGS+=(UNAME=Windows)
-    FLAGS+=(LDFLAGS="${LDFLAGS} -L${libdir} -shared")
-else
-    FLAGS+=(UNAME="$(uname)")
-    FLAGS+=(LDFLAGS="${LDFLAGS} -L${libdir}")
-fi
-
-BLAS_NAME=blastrampoline
-if [[ ${nbits} == 64 ]]; then
-    SUN="-DSUN64 -DLONGBLAS='long long'"
-fi
-
-FLAGS+=(BLAS="-l${BLAS_NAME}" LAPACK="-l${BLAS_NAME}")
-
-# Disable METIS in CHOLMOD by passing -DNPARTITION and avoiding linking metis
-#FLAGS+=(MY_METIS_LIB="-lmetis" MY_METIS_INC="${prefix}/include")
-FLAGS+=(UMFPACK_CONFIG="$SUN" CHOLMOD_CONFIG+="$SUN -DNPARTITION" SPQR_CONFIG="$SUN")
-
-make -j${nproc} -C SuiteSparse_config "${FLAGS[@]}" library config
-
 for proj in SuiteSparse_config AMD BTF CAMD CCOLAMD COLAMD CHOLMOD LDL KLU UMFPACK RBio SPQR; do
-    make -j${nproc} -C $proj "${FLAGS[@]}" library CFOPENMP="$CFOPENMP"
-    make -j${nproc} -C $proj "${FLAGS[@]}" install CFOPENMP="$CFOPENMP"
+    cd ${proj}/build
+    cmake .. \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=${prefix} \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DNOPENMP=ON \
+        -DNPARTITION=ON \
+        -DBLAS_FOUND=1 \
+        -DBLAS64_SUFFIX="_64" \
+        -DBLAS_LIBRARIES="${libdir}/libblastrampoline.${dlext}" \
+        -DBLAS_LINKER_FLAGS="blastrampoline" \
+        -DBLA_VENDOR="${BLAS_NAME}" \
+        -DALLOW_64BIT_BLAS=ON \
+        -DLAPACK_FOUND=1 \
+        -DLAPACK_LINKER_FLAGS="blastrampoline" \
+        -DLAPACK_LIBRARIES="${libdir}/libblastrampoline.${dlext}" \
+        -DBLAS_UNDERSCORE \
+        DCMAKE_Fortran_COMPILER=""
```

Comment:
```suggestion
        -DCMAKE_Fortran_COMPILER=""
```
Missed the hyphen making it a CMake argument.